docs/variants/talos_2/overview.md

```diff
@@ -0,0 +1,15 @@
+# Overview
+
+* [Releases](releases.md) - list of all releases
+* [Building manual](building-manual.md) - how to build Dasharo compatible with
+  the Talos II
+* [Initial deployment](initial-deployment.md) - how to install Dasharo
+  firmware on the Talos II
+* [Hardware configuration matrix](hardware-matrix.md) - definition of hardware
```

Comment:
`Firmware update` and `Recovery` sections are missing

Reply:
Sections added